app/assess/routes.py

```diff
@@ -24,6 +28,35 @@ def funds():
     return render_template("funds.html", funds=funds)
 
 
+@assess_bp.route("/landing/", methods=["GET"])
+def landing():
```

Comment:
Might need some comments so future generations understand the mechanics of this :)

Reply:
Add docstrings and coments